src/v/kafka/server/group.cc

```diff
@@ -2669,4 +2669,15 @@ void group::add_pending_member(
     res.first->second.arm(timeout);
 }
 
+std::ostream& operator<<(std::ostream& o, const group::offset_metadata& md) {
+    fmt::print(
+      o,
+      "{log_offset:{}, offset:{}, metadata:{}, committed_leader_epoch:{}}",
```

Comment:
@VadimPlh @mmaslankaprv 

any literal `{` or `}` needs to be escaped in the format string. this was also backported, so a fix will need to be backported too.
 
```
@start: failed to log message: fmt='{}:{} - {log_offset:{}, offset:{}, metadata:{}, committed_leader_epoch:{}}': fmt::v7::format_error (argument not found)
```